components/Pagination/Pagination.tsx

```diff
@@ -20,29 +18,18 @@ export const Pagination = ({
   nextPage,
 }: PaginationProps) => {
   return (
-    <div className={Styles.container}>
+    <div className="mt-8 mb-5 flex flex-col flex-wrap items-center justify-center gap-3 sm:flex-row">
```

Comment:
```suggestion
    <nav className="mt-8 mb-5 flex flex-col flex-wrap items-center justify-center gap-3 sm:flex-row">
```

Reply:
Czy `role="navigation"` jest wymagane dla tagu `<nav>`? Z tego co widzę na mdn to taka rola jest domyślna dla tego elementu.

Reply:
Masz rację 🙂